bluesky/examples.py

```diff
@@ -4,6 +4,12 @@
 from threading import RLock
 import numpy as np
 from .run_engine import Msg
+from uuid import uuid4
+import uuid
+from tempfile import mkdtemp
+import os
+from bluesky.utils import new_uid, short_uid
+from filestore.handlers_base import HandlerBase
```

Comment:
Bluesky does not have a hard dependency on filestore. You probably don't need anything in `HandlerBase`, anyway, right?